benchmarks/versus-external-pools/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "poolifier-benchmarks",
+  "version": "1.0.0-internal",
+  "description": "This project contains benchmarks for the most used and popular Node.js thread/cluster pools implementations",
+  "private": true,
+  "main": "index.js",
+  "author": "pioardi",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
We could add some script here so it's much easier and more guided how and what to use

Reply:
I would add all the instructions to the readme, I Will do that and from thrre se can evaluate of we Need also a script